src/Frontend/Context/Auth.js

```diff
@@ -0,0 +1,161 @@
+import axios from "axios";
+import { createContext, useContext, useReducer } from "react";
+import { AxiosInstance } from "../../AxiosInstance";
+
+//constants
+const loginObject = {
+  openLogin: false,
+  openRegister: false,
+  authState: {
+    login: false,
+    register: false,
+  },
+  user: null,
+  isLoggedIn: false,
+  isAuthenticated: false,
+  loading: false,
+  error: "",
+  registration: {
+    processInitiated: false,
+    email: null,
+    isRegistered: false,
+  },
+  AuthMessage: null,
+};
+const LOGIN = "LOGIN";
```

Comment:
You can move these Constants to the Other Folder.

Reply:
These will be at top of the file followed redux practices that particular reducer file constants will be at that particular file